notes.bash_completion

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+_notes_complete_notes() {
+    local configured_dir=${NOTES_DIRECTORY%/} # Remove trailing slashes
+    local notes_dir="${configured_dir:-$HOME/notes}"
+    local items=($(compgen -f "$notes_dir/$1"))
+    for item in "${items[@]}"; do
+        [[ -d $item ]] && item="$item/"
+        local filename=${item#$notes_dir/}
+        COMPREPLY+=("${filename%.md}")
+    done
```

Comment:
This doesn't handle spaces correctly. If you have "my note.md" in your notes folder then `notes find <TAB><TAB>` will show completions for `my` and `note`.

Also, it includes matches for hidden files. That's a general `notes` issue I'd like to fix everywhere when I get a minute, but if it's easy it'd be nice to have it working properly here from the start.